src/test/java/com/nerodesk/takes/doc/TkAddFriendTest.java

```diff
@@ -49,21 +50,24 @@
     /**
      * TkAddFriend can add a friend.
      * @throws Exception If fails.
-     * @todo #144:15min We should add an assertion where we will check if the
-     *  added friend is in fact present in doc.friends(). However, class
-     *  MkFriends is not yet implemented, so it returns nothing. When it is
-     *  implemented let's add that assertion to make this test more robust and
-     *  comprehensive.
      */
     @Test
     public void addsFriend() throws Exception {
```

Comment:
@alevohin checkout line 64 below. It calls `act()` and then it checks the underlying `doc.friends()` as requested in the puzzle.